### ISSUE TITLE
USWDS - a11y: Fix color and scrollable region issues on Grid page.

### ISSUE THREAD

**Issue by mejiaj** [uswds/uswds-site#1111]
## Description

Related to #1104.

**id:** 'color-contrast',
**impact:** 'serious',
**tags:** [ 'cat.color', 'wcag2aa', 'wcag143' ],
**description:** 'Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds',
**help:** 'Elements must have sufficient color contrast',
**helpUrl:** 'https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=axeAPI',

**id:** 'scrollable-region-focusable',
**impact:** 'moderate',
**tags:** [ 'wcag2a', 'wcag211' ],
**description:** 'Elements that have scrollable content should be accessible by keyboard',
**help:** 'Ensure that scrollable region has keyboard access',
**helpUrl:** 'https://dequeuniversity.com/rules/axe/3.5/scrollable-region-focusable?application=axeAPI',

### Pages to fix
- [ ] Grid

## Steps to reproduce the issue

1. Open `run-axe.js`
2. Add component URLs from #1104 to `const PAGES` array
4. `npm run axe`

**Comment by mejiaj** [comment]
This was resolved with the big round of a11y updates related to #1227.
